### PR TITLE
MNT: Officially retire cubeviz

### LINKED ISSUE #598: Problem with testing the last release in Python 3.9
`pip install cubeviz` downgrades `pyqt5`, causing this message to appear when `mosviz` is also installed: `ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts. mosviz 0.3.1 requires pyqt5==5.12.3, but you have pyqt5 5.8 which is incompatible.`

Running `cubeviz` command gave me this error in a clean `conda` env with Python 3.9 after I installed `qt` from `conda`. I failed to install PySide from both conda and pip; not sure why.

```
$ cubeviz
Traceback (most recent call last):
  File ".../python3.9/site-packages/qtpy/__init__.py", line 204, in <module>
    from PySide import __version__ as PYSIDE_VERSION  # analysis:ignore
ModuleNotFoundError: No module named 'PySide'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File ".../cubeviz", line 5, in <module>
    from cubeviz.cubeviz import main
  File ".../cubeviz/__init__.py", line 38, in <module>
    from . import keyboard_shortcuts
  File ".../cubeviz/keyboard_shortcuts.py", line 2, in <module>
    from qtpy import QtCore, QtWidgets, QtGui, compat
  File ".../qtpy/__init__.py", line 210, in <module>
    raise PythonQtError('No Qt bindings could be found')
qtpy.PythonQtError: No Qt bindings could be found
```

### PR DESCRIPTION
**Pull request by pllim**
Fix #598 (in a way)

Pre-requisite for #594 

With this patch:

```
$ cubeviz
cubeviz is no longer supported, please use jdaviz. If you must use cubeviz, please try the 0.3.x series in Python 3.6.
```

```python
>>> import cubeviz
cubeviz is no longer supported, please use jdaviz. If you must use cubeviz, please try the 0.3.x series in Python 3.6.
```

### CODE REVIEW

File: cubeviz/__init__.py [hunk 1]
```diff
@@ -3,36 +3,14 @@
...
-from .flux_equivalences import CustomFluxEquivalences
-# We override the units.equivalencies.spectral_density function with
-# CustomFluxEquivalences before the program starts. We expect all libraries
-# to access CustomFluxEquivalences when calling for units.equivalencies.spectral_density
-units.equivalencies.spectral_density = CustomFluxEquivalences(units.equivalencies.spectral_density)
-units.spectral_density = units.equivalencies.spectral_density
-
-from . import keyboard_shortcuts
+print('cubeviz is no longer supported, please use jdaviz. '
+      'If you must use cubeviz, please try the 0.3.x series '
```
Comment (eteq) [inline cubeviz/__init__.py:15]:
```suggestion
      'If you must use legacy cubeviz, please try v0.3 '
```

**Review by eteq — COMMENTED** [review]
One minor suggestion since I yanked all but 0.3 in the series.

Also, I wonder if this PR should delete all the *other* files in cubeviz as well, since they aren't accessible anyway after this PR?

**Comment by pllim** [comment]
> delete all the other files

I debated with myself about it... and thought as long as nothing breaks on import, maybe not worth the trouble... 🤷

**Comment by pllim** [comment]
p.s. After merge, I will use this PR as a "template" for legacy `mosviz` and `specviz`.

**Comment by pllim** [comment]
Looks like you never set up RTD PR build here. Just trust me the doc will build then...

**Review by eteq — APPROVED** [review]
I heartily approve the use of :fire: in the commit message!

**Pull request merged**